src/app.scss

```diff
@@ -0,0 +1,44 @@
+#root {
+  height: 100%;
+  min-height: 850px;  
+  background: linear-gradient(0deg, #1CB5E0 20%, #050b45 100%) no-repeat;
+}
+
+:root, 
+.App {
+  text-align: center;
+
+  @keyframes anvil {
+    0% {
+      box-shadow: 0 0 0 rgba(241, 241, 241, 0);
+      opacity: 0;
+      transform: scale(1) translateY(0px);
+    }
+    1% {
+      box-shadow: 0 0 0 rgba(241, 241, 241, 0);
+      opacity: 0;
+      transform: scale(0.96) translateY(10px);
+    }
+    100% {
+      box-shadow: 0 0 500px rgba(241, 241, 241, 0);
+      opacity: 1;
+      transform: scale(1) translateY(0px);
+    }
+  }
+  .popup-content {
+    width: auto;
+    max-width: 95%;
+    -moz-transition: anvil 0.3s cubic-bezier(0.38, 0.1, 0.36, 0.9) forwards;
```

Comment:
let's use https://www.npmjs.com/package/autoprefixer instead of adding prefixes manually each time we might need them